### DISCUSSION TITLE
麻烦看一下豆瓣APP的开屏广告

### DISCUSSION THREAD

**Discussion by ylliatte** [AIsouler/GKD_subscription#262] [General]
之前李跳跳也无法跳过这个，GKD现在也不行。能研究一下吗？
![Screenshot_2024-06-28-16-22-17-87_a3a62bdaa42478dfcfd65cf54dff2719](https://github.com/AIsouler/GKD_subscription/assets/141936009/91343eae-e47e-4492-97cd-95c933abfaa0)

**Comment by AIsouler** [comment]
没有快照我怎么研究？

**Reply by ylliatte** [reply]
不好意思是我搞错了，可以屏蔽的。
